nevergrad/functions/images/core.py

```diff
@@ -29,34 +33,131 @@ def __init__(self, problem_name: str = "recovering", index: int = 0) -> None:
         # Storing data necessary for the problem at hand.
         assert problem_name == "recovering"  # For the moment we have only this one.
         assert index == 0  # For the moment only 1 target.
```

Comment:
In your case, Laurent, there is no "assert index == 0"; index is the index of the image to be attacked.

Reply:
I'm not sure we are ready yet for actual experiments from this. Cf my comments.
I recommend merging if this work for Laurent, and working on the testbed classmethod to provide support for experiments.

Reply:
(once the test work, i.e. after updating or removing the code in experiments.py and merging master)